command/command.go

```diff
@@ -135,118 +157,70 @@ func PrintableCommandArgs(isQuoteFirst bool, fullCommandArgs []string) string {
 	return strings.Join(cmdArgsDecorated, " ")
 }
 
-// RunCmdAndReturnExitCode ...
-func RunCmdAndReturnExitCode(cmd *exec.Cmd) (exitCode int, err error) {
+// Deprecated: Use Command instead.
+func runCmdAndReturnExitCode(cmd *exec.Cmd) (exitCode int, err error) {
 	err = cmd.Run()
 	exitCode = cmd.ProcessState.ExitCode()
 	return
 }
 
-// RunCmdAndReturnTrimmedOutput ...
-func RunCmdAndReturnTrimmedOutput(cmd *exec.Cmd) (string, error) {
```

Comment:
Made internal because it was [unused](https://github.com/search?q=org%3Abitrise-io+org%3Abitrise-steplib+%22RunCmdAndReturnTrimmedOutput%22&type=Code) in external repositories.

Reply:
Can you please check bitrise-io and bitrise-steplib orgs too? Not sure if GitHub search indexes not recently touched repos so the safest way is to check out all step repos and search in the local file system.
If you want to be extra sure the few ~10 3rd party golang steps may be checked for usages.

Reply:
I've seen search results in repositories that were archived years ago. For example, [this query](https://github.com/search?q=org%3Abitrise-io+RunCommandAndReturnStdout&type=code) returns `bitrise-bridge`, which was archived in 2019.

Do you think that counts?

Reply:
Discussed offline, we are confident the changes are correct.